easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-foss-2017b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '5.0.0'
```

Comment:
`5.0.1`?

Reply:
i just checked and its not available using the eb link for download yet, i did it anyway